RandLAPACK/misc/rl_util.hh

```diff
@@ -475,20 +475,23 @@ void rl_orhr_col(
 }
 
 template <typename T>
-// Function returns current date
-std::string getCurrentDate() {
+// Function returns current date and time
+std::string getCurrentDateTime() {
```

Comment:
I don't think we have free functions with the camelCase convention. Switch to snake_case.

Reply:
OMG IS THAT SOLID SNAKE REFERENCE
fixed